### PR TITLE
Simplified buffer usage naming (+ fluid cleanup)

### PR DESCRIPTION
**Pull request by iwoplaza**

### CODE REVIEW

File: apps/typegpu-docs/src/content/examples/simulation/fluid-double-buffering.ts [hunk 1]
```diff
@@ -520,30 +527,54 @@ function obstaclesToConcrete(): Parsed<BoxObstacle>[] {
...
+runtime.onPlumChange(limitedBoxXPlum, () => {
+  obstacles[OBSTACLE_BOX].x = runtime.readPlum(limitedBoxXPlum);
   primary.applyMovedObstacles(obstaclesToConcrete());
-}
+});
 
-function setObstacleY(idx: number, y: number) {
-  obstacles[idx].y = y;
+runtime.onPlumChange(boxYPlum, () => {
+  obstacles[OBSTACLE_BOX].y = runtime.readPlum(boxYPlum);
```
Comment (mhawryluk) [inline apps/typegpu-docs/src/content/examples/simulation/fluid-double-buffering.ts:561]: this is a common pattern, where we read value of changed plum inside `onPlumChange` listener. I think we should consider one day allowing passing the listener as a function of the plum value `(newValue) => {(...)}`

Comment (iwoplaza) [reply apps/typegpu-docs/src/content/examples/simulation/fluid-double-buffering.ts:561]: Definitely doable. I will make a task for it. #162

File: apps/typegpu-docs/src/content/examples/simulation/fluid-with-atomics.ts [hunk 2]
```diff
@@ -59,28 +59,25 @@ const sizeBuffer = wgsl.buffer(vec2u).$name('size').$allowUniform();
 
 const viscosityBuffer = wgsl.buffer(u32).$name('viscosity').$allowUniform();
 
-const debugInfoBuffer = wgsl
-  .buffer(atomic(u32))
-  .$name('debug')
-  .$allowMutableStorage();
+const debugInfoBuffer = wgsl.buffer(atomic(u32)).$name('debug').$allowMutable();
```
Comment (reczkok) [inline apps/typegpu-docs/src/content/examples/simulation/fluid-with-atomics.ts]:
```suggestion
```

File: apps/typegpu-docs/src/content/examples/simulation/fluid-with-atomics.ts [hunk 3]
```diff
@@ -59,28 +59,25 @@ const sizeBuffer = wgsl.buffer(vec2u).$name('size').$allowUniform();
...
 
 const viscosityData = viscosityBuffer.asUniform();
-const currentStateData = currentStateBuffer.asReadonlyStorage();
+const currentStateData = currentStateBuffer.asReadonly();
 const currentStateVertex = currentStateBuffer.asVertex();
 const sizeData = sizeBuffer.asUniform();
-const nextStateData = nextStateBuffer.asMutableStorage();
-const debugInfoData = debugInfoBuffer.asMutableStorage();
+const nextStateData = nextStateBuffer.asMutable();
+const debugInfoData = debugInfoBuffer.asMutable();
```
Comment (reczkok) [inline apps/typegpu-docs/src/content/examples/simulation/fluid-with-atomics.ts]:
```suggestion
```

**Pull request merged**